content/developer/packaging/basics.zh.md

```diff
@@ -11,7 +11,7 @@ date = 2020-08-04T02:13:57.919Z
 我们使用如下几个工具维护 AOSC OS 软件包：
 
   - [Ciel](https://github.com/AOSC-Dev/ciel-rs/)
-      - 用于管理 systemd-nspawn(1) 容器。
```

Comment:
This is deliberate, `(1)` means the section in the man pages.  I.e. you can use `man 1 systemd-nspawn` to get the man page.

Thus IMO this shouldn't be changed.